manager-bundle/src/Routing/RouteLoader.php

```diff
@@ -14,11 +14,12 @@
 
 use Contao\ManagerPlugin\PluginLoader;
 use Contao\ManagerPlugin\Routing\RoutingPluginInterface;
+use Symfony\Bundle\FrameworkBundle\Routing\RouteLoaderInterface;
 use Symfony\Component\Config\Loader\LoaderInterface;
 use Symfony\Component\HttpKernel\KernelInterface;
 use Symfony\Component\Routing\RouteCollection;
 
-class RouteLoader
+class RouteLoader implements RouteLoaderInterface
```

Comment:
Should this be part of this PR?

Reply:
Yes. Route loaders that are neither tagged as `routing.router_loader` nor use the marker interface are deprecated and will no longer work in Symfony 5.